core/src/data/clientGeoJsonSource.cpp

```diff
@@ -20,6 +20,20 @@ const uint8_t indexMaxZoom = 18;
 const uint32_t indexMaxPoints = 100000;
 double tolerance = 1E-8;
 
+class Task : public TileTask {
```

Comment:
Yes, makes sense to have the default implementation for hasData() now. The initial version of that commit did also override TileTask.process() which is now gone.